docs/changes/extern/12716.bugfix.rst

```diff
@@ -0,0 +1 @@
+Images added from 'https://cdn.datatables.net/1.10.12/images/' 
```

Comment:
Don't think quotes are necessary. Also, would be nice to concisely explain why they were added. Thanks!
```suggestion
Images added from https://cdn.datatables.net/1.10.12/images/
```